README.md

```diff
@@ -55,10 +65,13 @@ and then follow the steps described in the [Carthage documentation](https://gith
 import HealthVault
 ```
 
+## Registering your applications
+All applications must be registered with the [HealthVault Application Configuration Center](https://go.microsoft.com/fwlink/?linkid=838954) before they can connect to the service. During registration, the ACC will request that you upload an appropriate certificate which will be used subsequently to secure communications between your app and the service. For more information on how to obtain an appropriate certificate, please see [Creating Key Pairs](https://docs.microsoft.com/en-us/healthvault/concepts/connectivity/creating-key-pairs). 
```

Comment:
Put ACC in parenthesis earlier after it's spelled out. Otherwise, this is the first occurrence of ACC and it's not clear what it means.